test/typehelpers/typehelpers.go

```diff
@@ -197,10 +209,11 @@ func AllMultihashesFromAdChainDepth(t *testing.T, ad *schema.Advertisement, lsys
 			if err != nil {
 				return err
 			}
-			_, mh, err := multihash.MHFromBytes(b)
-			if err != nil {
-				return err
-			}
+			mh := multihash.Multihash(b)
+			//_, mh, err := multihash.MHFromBytes(b)
+			//if err != nil {
+			//	return err
+			//}
```

Comment:
These look like comments from my test branch; please feel free to remove the commented codes. Thanks